app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="d-flex justify-content-center align-items-center vh-100">
+<div class="card shadow " >
+<div class="card-header">
+  <h2 class="card-title text-center">Sign in</h2>
+</div>
+
+<div class="card-body">
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="form-group my-3">
+   <%= f.label :email ,class:'label'%>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder:'email' ,class:'form-control form-control-lg'%>
+  </div>
+
+  <div class="form-group my-3">
+    <%= f.label :password ,class:'label'%>
+    <%= f.password_field :password, autocomplete: "current-password", placeholder:'password',class:'form-control form-control-lg' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="form-group my-3">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class:'btn btn-primary btn-lg'%>
+  </div>
```

Comment:
- [ ] Nice job making the signup/login page work 👌 However, this activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your login view:**        |  **Wireframe login view (Required):**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/ambrose-kibet/some_blog/assets/12745474/083306fa-09de-4f13-815f-294fa6227303"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">